src/NLog/Common/ConversionHelpers.cs

```diff
@@ -68,30 +68,30 @@ public static class ConversionHelpers
         /// Converts the string representation of the name or numeric value of one or more enumerated constants to an equivalent enumerated object. A parameter specifies whether the operation is case-sensitive. The return value indicates whether the conversion succeeded.
         /// </summary>
         /// <typeparam name="TEnum">The enumeration type to which to convert value.</typeparam>
-        /// <param name="value">The string representation of the enumeration name or underlying value to convert.</param>
-        /// <param name="result">When this method returns, result contains an object of type TEnum whose value is represented by value if the parse operation succeeds. If the parse operation fails, result contains the default value of the underlying type of TEnum. Note that this value need not be a member of the TEnum enumeration. This parameter is passed uninitialized.</param>
+        /// <param name="inputValue">The string representation of the enumeration name or underlying value to convert.</param>
+        /// <param name="resultValue">When this method returns, result contains an object of type TEnum whose value is represented by value if the parse operation succeeds. If the parse operation fails, result contains the default value of the underlying type of TEnum. Note that this value need not be a member of the TEnum enumeration. This parameter is passed uninitialized.</param>
         /// <returns><c>true</c> if the value parameter was converted successfully; otherwise, <c>false</c>.</returns>
         /// <remarks>Wrapper because Enum.TryParse is not present in .net 3.5</remarks>
-        internal static bool TryParse<TEnum>(string value, out TEnum result) where TEnum : struct
+        internal static bool TryParse<TEnum>(string inputValue, out TEnum resultValue) where TEnum : struct
         {
-            return TryParse(value, false, out result);
+            return TryParse(inputValue, true, out resultValue);
```

Comment:
This is a bit unexpected, changing the default for the whole code?

Reply:
(I missed the intro text, oops)

Reply:
Enum.TryParse (.net 4+) is case sensitive by default?

Reply:
Well when parsing then one wants to be case-insensitive, unless knowing the writer is strict. So ignoreCase=true is a better default for NLog. Just like for the public method